### PR TITLE
fixed not enough data preventing activites view and moved it to graph…

### LINKED ISSUE #1690: Freshly created market Transaction/Activity on History not shown
Network: Rinkeby
Market: http://localhost:3000/#/0xae875887f2660c4d7a5559bb2e337e7d1bd3757c
history

### PR DESCRIPTION
**Pull request by Mi-Lan**
closes #1690 
- This is market on rinkeby which can be tested on http://localhost:3001/#/0xebfc260e432e413bdfe311b70706a3afe686e3ae
- I left the message just for graph since it looks like crap without enough data
Screenshot 2021-03-02 at 12 50 12

### CODE REVIEW

**Review by pimato — CHANGES_REQUESTED** [review]
Something happened with the borders here:
Bildschirmfoto 2021-03-02 um 13 57 12

**Comment by pimato** [comment]
Also the chart time picker buttons have way more space in between compared to my designs here:
https://www.figma.com/file/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=2182%3A3

**Comment by Mi-Lan** [comment]
> Also the chart time picker buttons have way more space in between compared to my designs here:
> https://www.figma.com/file/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=2182%3A3

@pimato i included here the improper spacing...

**Review by pimato — CHANGES_REQUESTED** [review]
onHover of the time item should have a darker text color:
Bildschirmfoto 2021-03-02 um 20 34 17

check out:
https://www.figma.com/file/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=2182%3A222

**Comment by Mi-Lan** [comment]
> onHover of the time item should have a darker text color:
> Bildschirmfoto 2021-03-02 um 20 34 17
> 
> check out:
> https://www.figma.com/file/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=2182%3A222

Screenshot 2021-03-03 at 00 58 44

Well for me they do have darker color?

**Review by hexyls — APPROVED** [review]

**Review by pimato — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (pRoy24) [inline yarn.lock:1]: This file needs to be removed

File: app/src/components/market/common/history_section/history_select.tsx [hunk 2]
```diff
@@ -336,6 +305,7 @@ export const History_select: React.FC<Props> = ({
       ) : (
         <HistoryChart
           data={data}
+          hasEnoughData={notEnoughData}
```
Comment (pRoy24) [inline app/src/components/market/common/history_section/history_select.tsx:308]: Should be notEnoughData={notEnoughData}

File: app/src/components/market/common/history_chart/index.tsx [hunk 3]
```diff
@@ -143,6 +159,9 @@ export const HistoryChart: React.FC<Props> = ({
   if (!data || status === 'Loading' || sharesDataLoader) {
     return <CustomInlineLoading message="Loading Trade History" />
   }
+  if (hasEnoughData) {
```
Comment (pRoy24) [inline app/src/components/market/common/history_chart/index.tsx:162]: hasEnoughData means it actually has data, this prop should be nonEnoughData or doesNotHaveEnoughData

File: app/src/components/market/common/history_chart/index.tsx [hunk 4]
```diff
@@ -99,10 +113,12 @@ type Props = {
   isScalar?: Maybe<boolean>
   sharesDataLoader: boolean
   status: any
+  hasEnoughData: any
```
Comment (pRoy24) [inline app/src/components/market/common/history_chart/index.tsx:116]: This popType should be boolean

**Review by pRoy24 — CHANGES_REQUESTED** [review]

File: app/src/components/market/common/history_section/history_select.tsx (hunk 2 above)
Comment (Mi-Lan) [reply app/src/components/market/common/history_section/history_select.tsx:308]: yeah totally messed up naming on that piece of data !

**Review by pRoy24 — APPROVED** [review]
LG

**Pull request merged**